build.gradle

```diff
@@ -24,25 +24,18 @@
 
 
 plugins {
-  id 'org.scm-manager.smp' version '0.7.2'
+  id 'org.scm-manager.smp' version '0.7.5'
 }
 
 dependencies {
   plugin "sonia.scm.plugins:scm-mail-plugin:2.1.0"
   optionalPlugin "sonia.scm.plugins:scm-editor-plugin:2.0.0"
   optionalPlugin "sonia.scm.plugins:scm-landingpage-plugin:1.0.0"
   testImplementation "com.github.spullara.mustache.java:compiler:0.9.7"
-
-  testImplementation "com.github.sdorra:shiro-unit:1.0.1"
-
-  testImplementation "org.jboss.resteasy:resteasy-validator-provider:4.5.8.Final"
-  testImplementation "org.hibernate.validator:hibernate-validator:6.1.6.Final"
-  testImplementation "javax.el:javax.el-api:3.0.0"
-  testImplementation "org.glassfish:javax.el:3.0.1-b11"
 }
 
 scmPlugin {
-  scmVersion = "2.15.0"
+  scmVersion = "2.15.2-SNAPSHOT"
```

Comment:
This would imply an incompatibility of the review plugin with 2.15.0 without technical need. I think it would be correct to remove the dependencies, when the new core version is needed due to other features, but I would decline this change just to fix the dependencies. This would be more so the case with other plugins depending on an SCM-Manager with even older versions.